source/MaterialXTest/MaterialXGenOgsXml/GenOgsXml.cpp

```diff
@@ -77,16 +74,7 @@ class OgsXmlShaderGeneratorTester : public GlslShaderGeneratorTester
             return false;
         }
 
-        if (mx::ShaderRefPtr shaderRef = element->asA<mx::ShaderRef>())
-        {
-            mx::MaterialPtr material = shaderRef->getParent()->asA<mx::Material>();
-            if (material && dumpMaterials.count(material->getName()))
-            {
-                std::ofstream file(material->getName() + ".xml");
-                file << fragment->getFragmentSource();
-                file.close();
-            }
-        }
+        sourceCode.push_back(fragment->getFragmentSource());
```

Comment:
Fix for test as there are no shaderrefs. Just push the source code back for all compiles.